parkhangs-webapp/src/features/modal/ModalDetail.css

```diff
@@ -77,10 +77,39 @@
 .AddEventForm {
     display: flex;
     flex-direction: column;
-    width: 50%;
+    width: 100%;
+    padding:0.5rem;
+    background-color:#E3F1DF;
 }
 
 .EventsList-populated{
     max-height: 31rem;
     overflow: scroll
 }
+
+@media only screen and (max-width:600px){
+    .modal-container {
+        padding-top: 0;
+        background-color: #E3F1DF;
```

Comment:
is there a reason you're changing the background color when the screen size changes

Reply:
Oh I was testing using different colors for the two queries initially, I can move it out of the queries if that bothers you